sky/packages/updater/lib/main.dart

```diff
@@ -67,14 +100,15 @@ class UpdateTask {
   String _dataDir;
   String _tempPath;
 
-  _readLocalManifest() async {
-    String manifestPath = path.join(_dataDir, kManifestFile);
-    String manifestData = await new File(manifestPath).readAsString();
-    _currentManifest = yaml.loadYaml(manifestData, sourceUrl: manifestPath);
+  Future _readLocalManifest() async {
+    String bundlePath = path.join(_dataDir, kBundleFile);
```

Comment:
It catches the case where you accidentally _do_ change it again, e.g. because you accidentally reuse the variable name. Mostly it's just that if we do it reliably across the entire codebase, then when people copy-paste our code into their projects, they're more likely to also do it, and then more likely to also catch their errors. The code we write will be used as the model for the code to write when writing Flutter apps, for better or worse.